### PR TITLE
[TASK][27064] User show my stores

### PR DESCRIPTION
**Pull request by nthtra-110273**
Screen Shot 2020-08-26 at 09 22 15
https://edu-redmine.sun-asterisk.vn/issues/27064

### CODE REVIEW

File: app/assets/stylesheets/store/index.scss [hunk 1]
```diff
@@ -5,3 +5,7 @@
 #slider {
   padding: 0;
 }
+
+#store_product {
+  display: flex;
+}
```
Comment (nguyennhathai) [inline app/assets/stylesheets/store/index.scss]: endline

File: app/views/stores/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+                          <h5><%= store.name %></h5>
+                          <p><%= store.description %></p>
+                        </div>
+                      </div>
+                    <% end %>
+                  <% end %>
+                </div>
+                <a class="carousel-control-prev" href="#carouselExampleCaptions" role="button" data-slide="prev">
+                  <span class="carousel-control-prev-icon" aria-hidden="true"></span>
+                  <span class="sr-only">Previous</span>
```
Comment (nguyennhathai) [inline app/views/stores/index.html.erb]: i18n

File: app/views/stores/index.html.erb [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+            </div>
+          </div>
+          <div class="lead" style="margin-top: 20px;">
+            <h3><%= t ".experience_title" %></h3>
+            <p class="text__footer"><%= t ".experience" %><a href="#"> </a></p>
+          </div>
+        </div>
+      </div>
+    </section>
+  </main>
```
Comment (nguyennhathai) [inline app/views/stores/index.html.erb]: endline

File: app/views/stores/index.html.erb [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+      <ul class="ray-breadcrumbs">
+        <li class="ray-breadcrumb">
+          <%= t ".search" %>
+        </li>
+      </ul>
+    </section>
+    <section class="content container">
+      <div class="header">
+        <div class="header__right">
+          <h1 id="slot_name_title">My stores</h1>
```
Comment (nguyennhathai) [inline app/views/stores/index.html.erb]: i18n

File: app/controllers/stores_controller.rb [hunk 5]
```diff
@@ -0,0 +1,11 @@
+class StoresController < ApplicationController
+  before_action :logged_in_user, :find_user_by_id
+  def show
```
Comment (hieplt-0914) [inline app/controllers/stores_controller.rb:4]: cách 1 dòng ra

File: app/views/stores/index.html.erb [hunk 6]
```diff
@@ -0,0 +1,108 @@
...
+          <p> <%= t ".address" %> </p>
+        </div>
+      </div>
+      <div class="main">
+        <div class="main__left">
+          <div class="products" id="slot-info-products">
+            <% @stores.each do |store| %>
+              <%= link_to user_store_path(current_user) do %>
+                <div class="product slot-info-product" id="store_product">
+                  <%= image_tag class="slot-info-product-img"src="https://www.mlive.com/resizer/YEU5ywOU3--21qj7nAlYwDOT_64=/1280x0/smart/arc-anglerfish-arc2-prod-advancelocal.s3.amazonaws.com/public/4G27NNGGWJDYHIBIL2YJQ6DVPA.jpg" %>
```
Comment (hieplt-0914) [inline app/views/stores/index.html.erb]:
```suggestion
                  <%= image_tag class="slot-info-product-img" src="https://www.mlive.com/resizer/YEU5ywOU3--21qj7nAlYwDOT_64=/1280x0/smart/arc-anglerfish-arc2-prod-advancelocal.s3.amazonaws.com/public/4G27NNGGWJDYHIBIL2YJQ6DVPA.jpg" %>
```

File: app/views/stores/index.html.erb [hunk 7]
```diff
@@ -0,0 +1,108 @@
...
+            </div>
+            <div class="preview">
+              <li class="p-info-img fas fa-dollar-sign"></li>
+              <p><%= t ".affordable" %></p>
+            </div>
+            <div class="preview">
+              <li class="p-info-img fas fa-smile-wink"></li>
+              <p><%= t ".location" %></p>
+            </div>
+            <a href="https://bit.ly/2CS5Il7" target="_blank"><%= t ".googlemap_link" %></a>
```
Comment (hieplt-0914) [inline app/views/stores/index.html.erb]: link_to

File: app/views/stores/index.html.erb [hunk 8]
```diff
@@ -0,0 +1,108 @@
...
+                </a>
+                <a class="carousel-control-next" href="#carouselExampleCaptions" role="button" data-slide="next">
+                  <span class="carousel-control-next-icon" aria-hidden="true"></span>
+                  <span class="sr-only"><%= t ".next" %></span>
+                </a>
+            </div>
+          </div>
+          <div class="lead" style="margin-top: 20px;">
+            <h3><%= t ".experience_title" %></h3>
+            <p class="text__footer"><%= t ".experience" %><a href="#"> </a></p>
```
Comment (hieplt-0914) [inline app/views/stores/index.html.erb]: link_to

**Pull request merged**